llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -1737,39 +1776,14 @@ void RISCVFrameLowering::emitCalleeSavedRVVPrologCFI(
   for (auto &CS : RVVCSI) {
     // Insert the spill to the stack frame.
     int FI = CS.getFrameIdx();
```

Comment:
Is there a reason you didn't replace this loop with a call to `emitCFIforCSI` with a new `CFISaveRVVRegisterEmitter`? That would give a little better symmetry in those objects, even though it doesn't get rid of the calls to `emitCalleeSavedRVVPrologCFI` because of the other logic happening before this loop.

Reply:
(CFISaveRVVRegisterEmitter is now gone, so this question doesn't matter)